frontend/src/components/page-about/index/values.js

```diff
@@ -0,0 +1,89 @@
+import * as React from "react";
+import { Collapsible } from "../../collapsible";
+import {
+  container,
+  middleContainer,
+} from "../../../css/belief.module.css";
+import {
+  titleContainer,
+  title,
+} from "../../../css/values.module.css";
+
+const saltValues = [
+  {
+    sectionHead: "S - Spirit-led ministry",
+    sectionBody:
+      "To be connected with God and anointed by His Spirit, in order to hear His direction and step out in faith to minister to His people with His power.",
+  },
+  {
+    sectionHead: "L - Leadership development",
+    sectionBody:
+      "To develop people to their full potential and in Christ-like character in order to impact their spheres of influence and increase the spread of the Gospel to future generations.",
+  },
+  {
+    sectionHead: "T - Transculturalism",
+    sectionBody:
+      "To make the decision to go through discomforts and difficulties, in order to develop understanding and delight in people from a different culture.",
+  },
+];
+
+const commandValues = [
+  {
+    sectionHead: "C - Community",
+    sectionBody:
+      "To willingly come together as a diverse group of people to experience and live out the Gospel, in order to fulfill God's purposes.",
+  },
+  {
+    sectionHead: "M - Ministry inside and outside",
+    sectionBody:
+      "To be a visible display and viable demonstration of God's Kingdom by stewarding our God-given spiritual gift(s), resources and experiences to build God's Church and transform the world.",
+  },
+  {
+    sectionHead: "M - Missions through church planting",
+    sectionBody:
+      "To obey and fulfill the Great Commission through the local church, which God has established so that we can be witnesses and servants of His redemptive purpose.",
+  },
+  {
+    sectionHead: "D - Discipleship",
+    sectionBody:
+      "To invite people to join us in the lifestyle of the Kingdom by imitating Christ, teaching people to obey Him, and imparting Biblical values.",
+  },
+];
+
+const Values = () => (
+  <div className={container}>
+    <div className={titleContainer}>
+      <div className="subheading">Our Values</div>
+      <div className={title}>
+        <h2>S</h2>
+        <h3>a</h3>
+        <h2>LT C</h2>
+        <h3>o</h3>
+        <h2>MM</h2>
+        <h3>an</h3>
+        <h2>D</h2>
```

Comment:
this is not proper way to use H# tags.  you use one only the whole heading and use span with different classes to give the effect we want. Could you rework this? This will definitely hurt our SEO (Search Enging Optimization).  Google will definitely not like this when they crawl our website.  Could you rethink a better way to do this? please

Reply:
you are right, ive pushed up a change to address this.